src/v2/components/settings/SettingsModal.tsx

```
@@ -0,0 +1,138 @@
+          <nav className="flex px-6" aria-label="Tabs">
+            {tabs.map(({ id, label, icon: Icon }) => (
+              <button
+                key={id}
+                onClick={() => setActiveTab(id)}
+                className={`relative py-4 px-1 flex items-center text-sm font-medium ${
+                  activeTab === id
+                    ? 'text-indigo-600 dark:text-indigo-400'
+                    : 'text-gray-500 hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-300'
+                } mr-8`}
+              >
+                <Icon className="h-5 w-5 mr-2" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)